### PR TITLE
Added utility method to remove forwarded port

### PR DESCRIPTION
**Pull request by sravanmedarapu**
@jlipps @imurchie

### CODE REVIEW

**Comment by imurchie** [comment]
Is this something we need to be able to do? Looks good, other than my not understanding the use case.

File: test/functional/adb-commands-e2e-specs.js [hunk 1]
```diff
@@ -85,6 +85,10 @@ describe('adb commands', function () {
   it('should forward the port', async () => {
     await adb.forwardPort(4724, 4724);
   });
+  it('should remove forwarded port', async () => {
+    await adb.forwardPort(8200, 6790);
+    await adb.removePortForward(8200);
```
Comment (jlipps) [inline test/functional/adb-commands-e2e-specs.js]: any way to add an actual verification here?

**Comment by sravanmedarapu** [comment]
This utility is required as part of UiAutomator2 driver implementation reported [issue](https://github.com/appium/appium-uiautomator2-driver/issues/47).
I am adding a check to forward the port to available not used port number and make sure we are removing the forwarded port to make it available for next session to forward.

File: test/functional/adb-commands-e2e-specs.js (hunk 1 above)
Comment (sravanmedarapu) [inline test/functional/adb-commands-e2e-specs.js]: Added verification checks.

**Comment by imurchie** [comment]
I'm good with this.

**Comment by imurchie** [comment]
Can you squash into a single commit, @sravanmedarapu ?

**Comment by sravanmedarapu** [comment]
Sure.

**Comment by jlipps** [comment]
@sravanmedarapu for some reason CI can't find the emulator. if you tell me you've run the functional tests and they pass with this change, I'll go ahead and merge/publish

**Comment by sravanmedarapu** [comment]
I ran the functional tests, they are passing :thumbsup:. I think we are good to go.

**Comment by jlipps** [comment]
published as appium-adb 2.7.0

**Pull request merged**